### PR TITLE
A few more typos/possible infelicities

### PR DESCRIPTION
**Pull request by cormullion**
I went through some more chapters (not the data_vis chapters) and edited out a few typos/possible infelicities. Hope they're acceptable to you and useful.

### CODE REVIEW

**Review by storopoli — APPROVED** [review]
For the `stats` and `dataframes_performance` sections/chapter I agree. Thanks a lot @cormullion. I would also ask for @rikhuijzer and @lazarusA to review the changes in the sections that they authored.

**Comment by lazarusA** [comment]
After printing out the data vis chapter today. I will rewrite that chapter, explaining where necessary how to build the more complicated figures. So, @cormullion if you are planning to take a look a that also later, please hold on a little bit. By end of the week I will have it.

File: contents/dataframes_load_save.md [hunk 1]
```diff
@@ -93,7 +93,7 @@ end # hide
 """)
 ```
 
-Text file formats. like CSV and TSV files, can also come up with other delimiters, such as semicolons ";", spaces "\ ", or even something as unusual as "π".
+Text file formats like CSV and TSV files can also be found that use other delimiters, such as semicolons ";", spaces "\ ", or even something as unusual as "π".
```
Comment (rikhuijzer) [inline contents/dataframes_load_save.md:96]: This, for some reason, sounds very British. I see that as a good thing :yum:

File: contents/notation.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ Some of these rules might sound pedantic, but we found that they make the code m
 From the style guide, we attempt to adhere specifically to:
 
 - At most 92 characters per line in code (in Markdown files, longer lines are allowed).
-- With `using`, import at most one module per line
+- With `using`, import at most one module per line.
```
Comment (rikhuijzer) [inline contents/notation.md:29]: I got confused by this sentence myself. 

```suggestion
- When loading code via `using`, load at most one module per line.
```

**Review by rikhuijzer — APPROVED** [review]

File: contents/notation.md (hunk 2 above)
Comment (rikhuijzer) [reply contents/notation.md:29]: Nevermind, I'll take care of this.

**Comment by rikhuijzer** [comment]
@cormullion Thanks a lot again!

**Pull request merged**